products/quasar.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Quasar
+category: framework
+tags: javascript-runtime
+iconSlug: quasar
+permalink: /quasar
+versionCommand: quasar -v
+releasePolicyLink: https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule
+changelogTemplate: https://github.com/quasarframework/quasar/releases/tag/quasar-v__LATEST__
+activeSupportColumn: true
```

Comment:
```suggestion
activeSupportColumn: true
eolColumn: LTS support
```

So that the `Security support` column has the same name as in the Quasar documentation.